src/nrnpython/nrnpy_p2h.cpp

```diff
@@ -49,7 +49,7 @@ static void call_python_with_section(Object* pyact, Section* sec) {
     if (!r) {
         char* mes = nrnpyerr_str();
         if (mes) {
-            Fprintf(stderr, "%s\n", mes);
+            std::cerr << mes << std::endl;
```

Comment:
It's not obvious from the definition of `Fprinf` but it does serve a purpose and using `std::cerr` and `std::cout` isn't ideal. See #3052.